libs/sdk-backend-tiger/src/backend/tigerSpecificFunctions.ts

```diff
@@ -379,6 +403,19 @@ export type TigerSpecificFunctions = {
     ) => Promise<IInvitationUserResponse>;
     getWorkspaceDataFiltersLayout?: () => Promise<WorkspaceDataFiltersLayout>;
     setWorkspaceDataFiltersLayout?: (workspaceDataFiltersLayout: WorkspaceDataFiltersLayout) => Promise<void>;
+    getWorkspaceDataFilters?: (workspaceId: string, objectId: string) => Promise<WorkspaceDataFiltersResult>;
+    setWorkspaceDataFilters?: (
+        workspaceId: string,
+        workspaceDataFilter: WorkspaceDataFilter,
+    ) => Promise<WorkspaceDataFiltersResult>;
+    getWorkspaceDataFilterSettings?: (
+        workspaceId: string,
+        objectId: string,
+    ) => Promise<WorkspaceDataFilterSettingsResult>;
+    setWorkspaceDataFilterSettings?: (
+        workspaceId: string,
+        workspaceDataFilterSettings: WorkspaceDataFilterSettings,
+    ) => Promise<WorkspaceDataFilterSettingsResult>;
```

Comment:
As also mentioned in the other comment, if there is no need to specify the return type for a `setXXX` function. Please change the return type to `Promise<void>`.